# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hugin/wx.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- hugin-0.6/src/hugin/PreviewFrame.cpp_orig	2006-08-14 22:39:08.930715250 +0200
+++ hugin-0.6/src/hugin/PreviewFrame.cpp	2006-08-14 22:39:40.808707500 +0200
@@ -551,7 +551,7 @@
 #else
     // This is a bad call.. we just want to repaint the window, instead we will
     // process user events as well :( Unfortunately, there is not portable workaround...
-    wxYield();
+    //wxYield();
 #endif
 }
 
